benchkit/commandattachments/tcpdump.py

```diff
@@ -0,0 +1,94 @@
+# Copyright (C) 2024 Vrije Universiteit Brussel. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+import pathlib
+from typing import List
+
+from benchkit.benchmark import RecordResult, WriteRecordFileFunction
+from benchkit.platforms import Platform, get_current_platform
+from benchkit.shell.shellasync import AsyncProcess
+from benchkit.utils.types import PathType
+
+
+class TcpDump:
+
+    def __init__(
+        self,
+        interface: str = "",
+        filter_expression: str = None,
+        read_from_file: str = "",
+        show_timestamp: bool = False,
+        verbose_output: bool = False,
+        write_to_file: str = "",
+        snapshot_length: int = 0,
+        no_promiscuous_mode: bool = False,
+        display_packet_data: bool = False,
+        platform: Platform = None,
+    ) -> None:
+        self.interface = (interface,)
```

Comment:
this line is causing trouble, what are you trying to do?

Reply:
I think I did it in case multiple interfaces should be captured. I removed it.